.github/workflows/e2e-preset-test.yml

```diff
@@ -134,34 +154,6 @@ jobs:
       - name: 'Set ACR Subscription'
         run: az account set --subscription ${{secrets.AZURE_SUBSCRIPTION_ID}}
 
-      - name: Check if Image exists in ACR
-        id: check_image
```

Comment:
Why remove this portion - this has been helpful when images fail to upload and run needs to be restarted

Reply:
The original idea was because I play a trick here. "name" was changed to the test case name, not the image name due to implementation. For base image, we may have some extra test cases.

You are right. I rollback this change. Add an explicit field 'image-name'. I will still check the image existence here.